### ISSUE TITLE
Adding .mlpackage file into the ios example project

### ISSUE THREAD

**Issue by yohanessatria** [google/mediapipe#3741]
So I've been trying to add a .MLPackage file to the HolisticTrackingGPU ios project. The problem is somehow the target membership for all the example projects are disabled (greyed out), thus I can't link the .mlpackage file to the holistictrackinggpu and the xcode can't can't auto generate the model class. I tried to add the model class manually by :
1. add the model into a new blank project
2. let xcode generate the model class
3. copy all the code from the model class
4. and create a new header file for the holistictrackinggpu
5. paste all the copied data into the newly made header file
6. add the header file to the BUILD file
7. add a line of code to initialize the class in the view controller

after completing all the mentioned steps above and run the project, I got an error in the build file (Linking mediapipe/examples/ios/holistictrackinggpu/HolisticTrackingGpuApp_bin failed: (Aborted): wrapped_clang_pp failed: error executing command ). Interestingly, when I remove step 7, I have no error. I wonder what is missing from my build file because when I use the same model and initialize it in the other project without bazel, the project runs fine.

ViewController
<img width="622" alt="Screen Shot 2022-09-30 at 10 32 50 AM" src="https://user-images.githubusercontent.com/35787092/193177912-94d3f420-d3a3-4e76-a74a-a1a8d3adb2c5.png">

Build File
<img width="766" alt="Screen Shot 2022-09-30 at 10 34 40 AM" src="https://user-images.githubusercontent.com/35787092/193177918-725c9b88-8ead-4f20-97ae-2f728bcccc68.png">

So my question is how to properly add a .mlpackage file in the example project so that this project can be run properly.

Thanks

**Comment by yohanessatria** [comment]
Hi. Is there any solution on this? thanks

**Comment by yohanessatria** [comment]
hi @kuaashish

Have I written custom code: Yes
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): MacOS 12.5.1
Bazel version: 5.2.0
Xcode & Tulsi version (if issue is related to building in mobile dev environment): Xcode 14.0.1, Tulsi 0.20220310.88
Exact steps to reproduce:
for the detailed background on why I did this is in my first comment at the top.

In summary, I want to use the holistictrackinggpu example project to get the holistic keypoints from the user. I then want to use my custom model to predict what kind of sign language is performed by the user by utilizing these keypoints, but got an error on the BUILD file as shown in my previous comment, even though I already added the necessary files into the BUILD file.

Thanks for reaching me back :)
